modules/dr-restoring-cluster-state.adoc

```diff
@@ -285,6 +285,43 @@ If the status is `Pending`, or the output lists more than one running etcd pod,
 
 .. Repeat this step for each lost control plane host that is not the recovery host.
 
+. Remove the northbound database (nbdb) and southbound database (sbdb). Access the master nodes and recovery host by using SSH and run the following command:
```

Comment:
```suggestion
. Remove the northbound database (nbdb) and southbound database (sbdb). Access the recovery host and the remaining master nodes by using SSH and run the following command:
```

Reply:
Addressed - Changed the terminology for master nodes ( control plane nodes ) as per style guide rules.